packages/analyze-query/src/bin-analyze.ts

```diff
@@ -149,13 +149,12 @@ function runQuery(queryString: string): [number, number] {
       ]),
     ),
   };
-  suppressError.q = q;
-  suppressError.z = z;
 
-  eval(`q = z.query.${queryString};`);
+  const f = new Function('z', `return z.query.${queryString};`);
+  const q: Query<Schema, string, PullRow<string, Schema>> = f(z);
 
   const start = performance.now();
-  q.run();
+  await q.run();
```

Comment:
Maybe this was not really a problem because we use Promise.resolve for QueryImpl.